app/views/facilities/disputed_orders.html.haml

```diff
@@ -0,0 +1,12 @@
+= render "shared/transactions/headers"
+
+- content_for :h1 do
+  = current_facility
+
+- content_for :h2 do
+  %h2= t(".head")
+
+= content_for :top_block do
+  = render "shared/transactions/top", tab: "disputed_orders"
+
+= render "shared/transactions/table" if @order_details.present?
```

Comment:
We should show something if order details is empty. Take a look at what we do for some of the other search pages.